fetch_satnogs_db.py

```diff
@@ -0,0 +1,70 @@
+#This is a script that fetches transponder data from db.satnogs.org and parses it to flyby transponder data.
+import urllib2
+import json
+import sys
+from operator import itemgetter
+from os.path import expanduser
+
+# The script will overwrite the old database, user must agree to this.
+while True:
+    prompt = raw_input("This will overwrite the existing database at $HOME/.local/share/flyby/flyby.db, do you want to proceed? (yes/no): ")
+    if prompt == "yes" or prompt == "y" or prompt == "Yes": 
+        break
+    elif prompt == "no" or prompt == "n" or prompt == "No":
+        sys.exit("Script terminated")
+
+#Step 1: Fetch JSON transponder information from SatNOGS db.
+request = urllib2.urlopen("https://db.satnogs.org/api/transmitters")
+data = json.load(request)
+
+# Order data by norad_cat_id entry
+sorteddata = sorted(data, key=itemgetter('norad_cat_id'))
+
+#Step 2: Generate transponder database as expected by flyby.
+
+# Write to $HOME/.local/share/flyby/flyby.db
+with open(expanduser("~")+"/.local/share/flyby/flyby.db","w") as db:
+    previous_norad_id = False
+
+    # Iterate through SatNOGS database
+    for i in sorteddata:
```

Comment:
Fixed in commit commit 2c7cf97eb8319deadfac50773ebdc85e3b25b8c4

Renaming i to transponder